analyzers/fcc/Reader.py

```diff
@@ -51,6 +55,12 @@ class Reader(Analyzer):
     - event.gen_jets: gen jets
     - event.jets: reconstructed jets  
     '''
+
+##    def beginLoop(self, setup):
```

Comment:
remove? (on github, it looks like you have added a method that is commented out)